src/network/networkprotocol.h

```diff
@@ -597,24 +647,26 @@ enum ToClientCommand : u16
 			u8                spawner_flags
 			    bit 1: attractor_kill (particles dies on contact)
 			if attraction_mode > point {
-				tween<v3f> attractor_angle
-				u16        attractor_origin_attachment_object_id
+				tween<v3f> attractor_direction
+				u16        attractor_direction_attachment_object_id
 			}
 		}
 
 		tween<range<v3f>> radius
-		tween<range<v3f>> drag
 
-		u16 texpool_sz
-		texpool_sz.times {
+		u16 texpool_size
+		texpool_size.times {
 			u8 flags
 			-- bit 0: animated
-			-- other bits free & ignored as of proto v40
+			-- next bits: blend mode
```

Comment:
A reference to the corresponding enum name in C++ might be helpful here.

Reply:
Added.